src/Type/ParserNodeTypeToPHPStanType.php

```diff
@@ -58,6 +63,12 @@ public static function resolve($type, ?ClassReflection $classReflection): Type
 			}
 
 			return TypeCombinator::intersect(...$types);
+		} elseif ($type instanceof ConstFetch) {
```

Comment:
we added a lot of new cases here to support the different AST types, a default parameter can have, but was not yet supported.

is there some util I should call from here for this very basic conversion, or do I really need to else-if-case all possible classes here?